tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -646,6 +646,57 @@ public static function autoIncrementTypeMigrations(): iterable
             }
         }
     }
+
+    /**
+     * @param callable(AbstractSchemaManager):Comparator $comparatorFactory
+     *
+     * @dataProvider dataDropIndexInAnotherSchema
+     */
+    public function testDropIndexInAnotherSchema(
```

Comment:
Please consider the recommendation from https://github.com/doctrine/dbal/pull/6773#discussion_r1957223313.

Reply:
Done